rust/operator-binary/src/controller.rs

```diff
@@ -1023,18 +1027,25 @@ fn build_bundle_builder_start_command(merged_config: &OpaConfig, container_name:
     formatdoc! {"
         {COMMON_BASH_TRAP_FUNCTIONS}
         prepare_signal_handlers
-        /stackable/opa-bundle-builder{logging_redirects} &
+        mkdir -p {STACKABLE_LOG_DIR}/{container_name}
+        stackable-opa-bundle-builder{logging_redirects} &
         wait_for_termination $!
         ",
         // Redirects matter!
         // We need to watch out, that the following "$!" call returns the PID of the main (opa-bundle-builder) process,
         // and not some utility (e.g. multilog or tee) process.
         // See https://stackoverflow.com/a/8048493
+        // TODO: do we need multilog?
         logging_redirects = if console_logging_off {
-            format!(" &> >(/stackable/multilog s{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILE_SIZE_BYTES} n{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILES} {STACKABLE_LOG_DIR}/{container_name})")
+            format!(" &> {STACKABLE_LOG_DIR}/{container_name}/current")
         } else {
-            format!(" &> >(tee >(/stackable/multilog s{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILE_SIZE_BYTES} n{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILES} {STACKABLE_LOG_DIR}/{container_name}))")
+            format!(" &> >(tee {STACKABLE_LOG_DIR}/{container_name}/current)")
         },
+        // logging_redirects = if console_logging_off {
+        //     format!(" &> >(/stackable/multilog s{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILE_SIZE_BYTES} n{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILES} {STACKABLE_LOG_DIR}/{container_name})")
+        // } else {
+        //     format!(" &> >(tee >(/stackable/multilog s{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILE_SIZE_BYTES} n{OPA_ROLLING_BUNDLE_BUILDER_LOG_FILES} {STACKABLE_LOG_DIR}/{container_name}))")
+        // },
```

Comment:
Multilog is presumably used to avoid leaking space indefinitely. It's currently dropped since it's only packaged in the OPA product image, not the operator images. We _could_ copy it into the operator images too, but I'd suggest addressing https://github.com/stackabletech/operator-rs/issues/794 and https://github.com/stackabletech/operator-rs/issues/795 instead.

Reply:
Honestly I would leave the review of this part to @siegfriedweber and @xeniape as they are or have worked on this

Reply:
I think with this PR we do not use multilog anymore as well https://github.com/stackabletech/opa-operator/pull/555, so if we stop using it here as well we can remove it from the docker image?

Reply:
#555 still uses multilog, it's just wrapped by the process-logs script (see https://github.com/stackabletech/docker-images/pull/695/files#diff-71c6e1159cfa2bd30faedad64de9b509a5235759945cfcf0659db64567ffc083R135).

Reply:
damn, would actually have liked to get rid of that...its very old... :D

Reply:
Are we okay to live with the disk leak for now, or do we want to block on https://github.com/stackabletech/operator-rs/issues/795?

Reply:
After discussing with @siegfriedweber and @xeniape we're blocking this on https://github.com/stackabletech/operator-rs/issues/795.

Reply:
https://github.com/stackabletech/operator-rs/pull/814